pkg/internal/splitbytes/helpers.go

```diff
@@ -14,11 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package secretsmanager
+package splitbytes
```

Comment:
```suggestion
package bytes
```

I've probably misadvised you here, as i'm sure golangci_lint is going to complain about stuttering function names.

Should probably rename the package bytes, and the function split, so we have `bytes.Split(data []byte, maxSize int, iterFunc func([]byte))`

Reply:
Done. I'm surprised it doesn't count `bytes` as a protected namespace or something similar.